src/entities/Place/routes/getPlaceListById.ts

```diff
@@ -0,0 +1,24 @@
+import Context from "decentraland-gatsby/dist/entities/Route/wkc/context/Context"
+import ApiResponse from "decentraland-gatsby/dist/entities/Route/wkc/response/ApiResponse"
+import Router from "decentraland-gatsby/dist/entities/Route/wkc/routes/Router"
+
+import PlaceModel from "../model"
+import { getPlaceListQuerySchema } from "../schemas"
+import { GetPlaceListQuery } from "../types"
+
+export const validateGetPlaceListQuery = Router.validator<GetPlaceListQuery>(
+  getPlaceListQuerySchema
+)
+
+export const getPlaceListById = Router.memo(
+  async (ctx: Context<{}, "url" | "request" | "params" | "body">) => {
+    const placeIds = ctx.body as string[]
```

Comment:
To prevent a DoS, we must set a maximum amount of ids to be queried for. This will depend on how big the places response is.
Would you mind adding one?